changelog.md

```diff
@@ -66,6 +66,8 @@
 - `echo` and `debugEcho` will now raise `IOError` if writing to stdout fails.  Previous behavior
   silently ignored errors.  See #16366.  Use `-d:nimLegacyEchoNoRaise` for previous behavior.
 
+- Added `math.copySign`.
```

Comment:
how about create a math entry that groups all the additions: almostEqual, isNaN, copySign, + your upcoming PR's math additions? easier to read

Reply:
I will do this at the proper time.